### PR TITLE
create Home page

### PR DESCRIPTION
**Pull request by brunnermanu**
- Create first part of home page
- add second button
- add more styles
- add new tab "Articles"
- make navbar responsible (mobile view still not optimal)

screely-1673271618244

### CODE REVIEW

File: src/app/button-with-icon/button-with-icon.component.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import "src/styles/variables";
+@import "src/styles/maps";
+@import "src/styles/mixins";
+
+.button {
+	border: none;
+	background-color: $blue;
+	display: flex;
+	flex-wrap: nowrap;
```
Comment (khanlifts) [inline src/app/button-with-icon/button-with-icon.component.scss]: Do you need `flex-wrap`?

File: src/app/button-with-icon/button-with-icon.component.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	align-items: center;
+	gap: 10px;
+	border-radius: 4px;
+	padding: 10px 15px;
+	outline: none;
```
Comment (khanlifts) [inline src/app/button-with-icon/button-with-icon.component.scss]: same here for `outline`

File: src/app/button-with-icon/button-with-icon.component.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	transition: 0.2s ease-in-out;
+	text-align: center;
```
Comment (khanlifts) [inline src/app/button-with-icon/button-with-icon.component.scss]: same here for `text-align`

File: src/app/button-with-icon/button-with-icon.component.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@import "src/styles/variables";
+@import "src/styles/maps";
+@import "src/styles/mixins";
+
+.button {
```
Comment (khanlifts) [inline src/app/button-with-icon/button-with-icon.component.scss:5]: 13 lines do now repeat in the `button` and `button-with-icon` components. Do you want to move the styles into the map?

File: src/app/home/home.component.scss [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+.intro {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	@include page-styles(standard-spaces);
+	height: calc(100vh - 80px);
+	background-color: $extra-light;
+
+	&__content {
```
Comment (khanlifts) [inline src/app/home/home.component.scss]: Why did you commit this?

File: src/app/home/home.component.scss [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+	&__subtitle {
+		@include headings(p1);
+		border-left: 3px solid $dark;
+		padding-left: 6px;
+		margin: 30px 0;
+	}
+
+	&__email-input {
+		display: flex;
+		flex-wrap: nowrap;
```
Comment (khanlifts) [inline src/app/home/home.component.scss]: Do you need `flex-wrap: nowrap`?

File: src/app/home/home.component.scss [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+	border-radius: 4px;
+	padding: 10px 15px;
+	min-width: 250px;
+	@include headings(p2);
+}
+
+::placeholder {
+	@include headings(p2);
+	color: $gray
+}
```
Comment (khanlifts) [inline src/app/home/home.component.scss]:
```suggestion
  &::placeholder {
	  color: $gray
  }
}
```

File: src/app/navbar/navbar.component.scss [hunk 8]
```diff
@@ -1,50 +1,94 @@
...
-      &:hover {
-        color: $light-blue;
-      }
-    }
-
-    &--active {
-      color: $blue;
-    }
-  }
+	width: 100%;
```
Comment (khanlifts) [inline src/app/navbar/navbar.component.scss]: Do you need `width: 100%`?

File: src/app/navbar/navbar.component.scss [hunk 9]
```diff
@@ -1,50 +1,94 @@
...
+	height: 80px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: $extra-light;
+	box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.2);
+	position: relative;
+	z-index: 2;
```
Comment (khanlifts) [inline src/app/navbar/navbar.component.scss]: Does `z-index: 1;` suffice?

File: src/app/navbar/navbar.component.scss [hunk 10]
```diff
@@ -1,50 +1,94 @@
...
+		}
+	}
+
+	@include mobile {
+		&__logo {
+			margin-left: $layout-spacing-small;
+
+			&--logo-h1 {
+				@include headings(logo-h1);
+				font-size: 20px;
```
Comment (khanlifts) [inline src/app/navbar/navbar.component.scss]: Do you want to have a separate `headings-mobile` map so you don't have to add the `font-size`s each time?

File: src/app/navbar/navbar.component.scss [hunk 11]
```diff
@@ -1,50 +1,94 @@
...
+
+			&--logo-h2 {
+				@include headings(logo-h2);
+				font-size: 12px;
+			}
+		}
+		&__links {
+			margin-right: $layout-spacing-small;
+		}
+	}
```
Comment (khanlifts) [inline src/app/navbar/navbar.component.scss:67]: add empty space

File: src/app/navbar/navbar.component.scss [hunk 12]
```diff
@@ -1,50 +1,94 @@
...
+
+			&--logo-h2 {
+				@include headings(logo-h2);
+				font-size: 16px;
+			}
+		}
+		&__links {
+			margin-right: $layout-spacing-medium;
+		}
+	}
```
Comment (khanlifts) [inline src/app/navbar/navbar.component.scss:83]: add empty space

File: src/styles/variables.scss [hunk 13]
```diff
@@ -1,74 +1,94 @@
...
+		font-size: 64px,
+		color: $dark,
+		font-weight: 700,
+	),
+	h1-mobile: (
+		font-family: 'Plus Jakarta Sans',
+		font-size: 44px,
+		color: $dark,
+		font-weight: 700,
+	),
```
Comment (khanlifts) [inline src/styles/variables.scss:27]: You could move this to your new `heading-mobile-map`

File: src/app/button-with-icon/button-with-icon.component.scss (hunk 1 above)
Comment (brunnermanu) [reply src/app/button-with-icon/button-with-icon.component.scss]: No, I forgot that nowrap is default.

File: src/app/button-with-icon/button-with-icon.component.scss (hunk 4 above)
Comment (brunnermanu) [reply src/app/button-with-icon/button-with-icon.component.scss:5]: 👍

File: src/styles/variables.scss (hunk 13 above)
Comment (brunnermanu) [reply src/styles/variables.scss:27]: For now, I leave them in the heading-map and add mobile and tablet fonts in there.

**Pull request merged**